Source/Main.py

```diff
@@ -0,0 +1,8 @@
+import ImperatortoCK3Converter
+import Configuration
+
+ConfigurationFileName = input("Insert configuration filename: ")
+Configuration.readConfigurationFile(ConfigurationFileName)
+
+ImperatorSaveFileName = input("Insert save filename: ")
```

Comment:
don't prompt, use configuration.txt to get info. Youll get it there anyway by the frontend.

Reply:
I think I solved this in 755a825, could you check if that's what you meant?